packages/@mollitia/redis-storage/vite.config.ts

```diff
@@ -0,0 +1,10 @@
+// Helpers
+import { defineLibConfig } from '../../../shared/vite/index.js';
+import { version } from './package.json';
+
+export default defineLibConfig({
+  name: 'Mollitia',
```

Comment:
Looking at that, I wonder if this should be `MollitiaRedis` here (probably the same for `MollitiaPrometheus` that has the same issue.
IIRC, this is the name used for `umd` export, so we are conflicting with `window.Mollitia` here.